### PR TITLE
Randomize evolve speed (Closes #4619 )

### LINKED ISSUE #4619: [Feature Request] Randomize Evolve Speed
### Short Description

Randomize Evolve Speed

### Possible solution

"max_evolve_speed": 40,
"min_evolve_speed": 20,

### How it would help others

could make bot detection harder and bot more realistic.

### PR DESCRIPTION
**Pull request by Gzerox**
## Short Description:
This will allow to randomize evolve speed.

## Fixes/Resolves/Closes (please use correct syntax):
- #4619

### CODE REVIEW

File: pokemongo_bot/cell_workers/evolve_pokemon.py [hunk 1]
```diff
@@ -128,7 +131,7 @@ def _execute_pokemon_evolve(self, pokemon, cache):
             new_pokemon = inventory.Pokemon(evolution)
             inventory.pokemons().add(new_pokemon)
 
-            sleep(self.evolve_speed)
+            sleep(randint(self.min_evolve_speed, self.max_evolve_speed))
```
Comment (mjmadsen) [inline pokemongo_bot/cell_workers/evolve_pokemon.py]: Could we use uniform instead to add a bit more variance?

**Comment by mjmadsen** [comment]
Please add to init that evolve_speed is depreciated and update configuration_files.md.

**Comment by solderzzc** [comment]
CI build failed. [#154501646](https://travis-ci.org/PokemonGoF/PokemonGo-Bot/builds/154501646)

**Comment by mjmadsen** [comment]
@solderzzc It looks like the tes api wrapper failed. Am I missing something here?

**Comment by Gzerox** [comment]
Seems the build now passed, strange thing :\

File: pokemongo_bot/cell_workers/evolve_pokemon.py [hunk 2]
```diff
@@ -26,6 +29,9 @@ def _validate_config(self):
         if isinstance(self.evolve_all, basestring):
             self.evolve_all = [str(pokemon_name).strip() for pokemon_name in self.evolve_all.split(',')]
 
+        if 'evolve_speed' in self.config:
+            self.logger.warning("evolve_speed is deprecated, please use instead 'min_evolve_speed' and 'max_evolved_speed'.")
```
Comment (Gzerox) [inline pokemongo_bot/cell_workers/evolve_pokemon.py]: Is ok to log and check in this way ?

**Comment by solderzzc** [comment]
@mjmadsen don't know why it failed, maybe caused by the base commit already had CI issue, and was fixed by merge

File: pokemongo_bot/cell_workers/evolve_pokemon.py (hunk 2 above)
Comment (solderzzc) [inline pokemongo_bot/cell_workers/evolve_pokemon.py]: @Gzerox We have the same kinds of code to ensure notify users the configuration file format changes. So it's ok.

**Pull request merged**